src/Microsoft.R.Host.vcxproj

```diff
@@ -101,6 +101,7 @@
       <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <SDLCheck>false</SDLCheck>
       <AdditionalIncludeDirectories>..\lib\picojson;..\lib\websocketpp</AdditionalIncludeDirectories>
+      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
```

Comment:
Lack of entry means default value, which is "Debug multithreaded DLL" for Debug, and "Release multithreaded DLL" for Release. So any other option requires explicitly setting the property. Available options are all combinations of debug/release, single-threaded/multithreaded, and DLL/static.